### PR TITLE
packages(nextjs-plugin): add docs and release

### PR DESCRIPTION
**Pull request by notrab**
# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [x] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [x] 🔨 Refactoring
- [ ] 🧪 Test
- [x] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: packages/grafbase-nextjs-plugin/README.md [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+
+### Step 1
+
+Install `grafbase` as a `devDependency`:
 
 ```bash
-npm run dev
-# or
-yarn dev
+npm install -D grafbase
```
Comment (vitalikda) [inline packages/grafbase-nextjs-plugin/README.md:14]:
```suggestion
npm install -D grafbase @grafbase/nextjs-plugin
```

File: packages/grafbase-nextjs-plugin/src/index.ts [hunk 2]
```diff
@@ -13,21 +16,28 @@ const runGrafbase = () => {
...
+      (process.env['GRAFBASE_API_URL'] &&
+        (process.env['GRAFBASE_API_URL'].startsWith('http://localhost') ||
+          process.env['GRAFBASE_API_URL'].startsWith('http://127.0.0.1'))) ||
+      (process.env['NEXT_PUBLIC_GRAFBASE_API_URL'] &&
+        (process.env['NEXT_PUBLIC_GRAFBASE_API_URL'].startsWith(
+          'http://localhost'
+        ) ||
+          process.env['NEXT_PUBLIC_GRAFBASE_API_URL'].startsWith(
+            'http://127.0.0.1'
+          )))
```
Comment (vitalikda) [inline packages/grafbase-nextjs-plugin/src/index.ts:35]: Could we use something like?
```ts
const GRAFBASE_API_URL = process.env['GRAFBASE_API_URL'] || process.env['NEXT_PUBLIC_GRAFBASE_API_URL']
const hasLocalApiUrl = /^http:\/\/(localhost|127\.0\.0\.1)/.test(GRAFBASE_API_URL)
```

File: packages/grafbase-nextjs-plugin/README.md [hunk 3]
```diff
@@ -1,34 +1,49 @@
...
+
+Add the following to `.env`:
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+```bash
+GRAFBASE_API_URL=http://localhost:4000/graphql
+```
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+You can also prefix with `NEXT_PUBLIC_` if your Next.js app is to make requests from the client-side.
```
Comment (vitalikda) [inline packages/grafbase-nextjs-plugin/README.md:25]: I would just put it inside code snippet 
```bash
GRAFBASE_API_URL=http://localhost:4000/graphql
# OR
NEXT_PUBLIC_GRAFBASE_API_URL=http://localhost:4000/graphql
```

**Review by vitalikda — COMMENTED** [review]
Nice! 👍 
Left a few suggestions

File: packages/grafbase-nextjs-plugin/src/index.ts (hunk 2 above)
Comment (notrab) [reply packages/grafbase-nextjs-plugin/src/index.ts:35]: Yep. Great suggestion. I should have better refactored after dumping my previous POC.

File: packages/grafbase-nextjs-plugin/README.md (hunk 1 above)
Comment (notrab) [reply packages/grafbase-nextjs-plugin/README.md:14]: Haha good spot!

**Review by vitalikda — APPROVED** [review]
🙌

**Pull request merged**